docs/source/index.rst

```diff
@@ -28,6 +28,7 @@ Welcome to rez's documentation!
    context_bundles
    suites
    managing_packages
+   releasing_packages
```

Comment:
Considering that it's basically what someone needs to do after building a package, it should go in the "General" section. Releasing a package is not considered advanced, It's a basic operation.